logstash/logstash.conf

```diff
@@ -0,0 +1,30 @@
+# vim: ts=2:sw=2:et:
```

Comment:
We don't store editor specific configuration in git repo. Please replace with something like http://editorconfig.org/